src/components/cc-header-app/cc-header-app.js

```diff
@@ -208,58 +199,57 @@ export class CcHeaderApp extends LitElement {
    * @param {CcHeaderAppChangedProperties} changedProperties
    */
   willUpdate (changedProperties) {
-    if (changedProperties.has('statusAndCommitState') && this.statusAndCommitState.type === 'loading') {
+    if (changedProperties.has('state') && this.state.type === 'loading') {
       this._lastUserAction = null;
     }
   }
```

Comment:
@hsablonniere (or anyone else 😉) I'm not sure I understood the point of this one.

I thought the goal was to reset _lastUserAction but I'm still not sure, I feel like I'm missing something 🤔

Reply:
Yes, I think the idea is:

* you click on on of the buttons
* we trigger the action on the API but the state doesn't change immediatly
* because of that, we display a "ok we got your message" with the lastUserAction
* and when the state changes we reset this so we can display the message of the new state

Reply:
Perfect, thanks for answering :wink: